### PR TITLE
Update dbms comapring models

### PR DESCRIPTION
**Pull request by KRG17**
updated all dbms and networking graphs and saved them

### CODE REVIEW

File: compare_model/compare_graph/net_compare.csv
Comment (prabudhall) [inline compare_model/compare_graph/net_compare.csv:1]: This file should be inside compare_model/compare_graph/networking . 
Do the needful changes in the code as well.

**Review by prabudhall — CHANGES_REQUESTED** [review]
Changes Requested

File: compare_model/compare_graph/net_compare.csv
Comment (prabudhall) [reply compare_model/compare_graph/net_compare.csv:1]: Or is it outside the subjects folder?

File: compare_model/compare_graph/dbms_compare.csv
Comment (prabudhall) [inline compare_model/compare_graph/dbms_compare.csv:1]: The file below this i.e. compare_model/compare_graph/net_compare.csv is having only 34 lines. So if you have updated the questions then please re run the code for networking and update the result for it as well

**Review by prabudhall — CHANGES_REQUESTED** [review]
Changes requested

File: compare_model/compare_graph/dbms_compare.csv
Comment (KRG17) [reply compare_model/compare_graph/dbms_compare.csv:1]: Yes, i have updated teh net_comapre.csv file

File: compare_model/compare_graph/net_compare.csv
Comment (KRG17) [reply compare_model/compare_graph/net_compare.csv:1]: Yes it is outside of the folder

**Review by prabudhall — APPROVED** [review]

**Pull request merged**